packages/transforms/naming-convention/test/naming-convention.spec.ts

```diff
@@ -93,8 +90,6 @@ describe('namingConvention wrap', () => {
     expect(userTypeEnumType.getValue('Admin')).toBeUndefined();
     const adminValue = userTypeEnumType.getValue('ADMIN');
     expect(adminValue).toBeDefined();
-    // expect(adminValue.value).toBe('admin');
-    expect(printSchemaWithDirectives(newSchema)).toMatchSnapshot();
```

Comment:
Why do we remove the schema snapshot?

Reply:
Well, that's a mistake. Had trouble with that, but should be added again. I will take care of that.

Reply:
Has been added. Test failes in bare mode because the bare 'mode' transformer does not apply the enum name convention to default values for arguments.

I just added an additional test for this case and start to work on a by in the next days.

Reply:
I would prefer to skip the failing tests for now and create an issue for this. 
Then we would only have technical change in this pull request and fix the bug in another one.

What do you think about this?

Reply:
For now I mark the tests as skipped.

I will unskip them when solving the new issue: #6419